_schemaV2.graphql

```diff
@@ -12600,6 +12606,14 @@ type SaleArtwork implements ArtworkEdgeInterface & Node {
 
   # Singular estimate field, if specified
   estimateCents: Int
+  extendedBiddingEndAt(
+    format: String
+
+    # A tz database time zone, otherwise falls back to "X-TIMEZONE" header. See
+    # http://www.iana.org/time-zones,
+    # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+    timezone: String
```

Comment:
Does this all "just work" without adding more code related to formatting?

Reply:
yes this uses our `date` helper